src/test/kotlin/org/rust/ide/annotator/fixes/AddMutableFixText.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.annotator.fixes
+
+
+import org.rust.ide.inspections.RsBorrowCheckerInspection
+import org.rust.ide.inspections.RsInspectionsTestBase
+
+class AddMutableFixText : RsInspectionsTestBase(RsBorrowCheckerInspection()) {
+    fun `test add mutable fix to self`() = checkFixByText("Make `self` mutable", """
+        struct A {}
+
+        impl A {
+            fn foo(&mut self) {  }
+
+            fn bar (&self) {
+                <error>se/*caret*/lf</error>.foo();
```

Comment:
Let's put /*caret* marker before or after `self`. It's hard to read an exploded identifier.